lib/cubit/security/security_cubit.dart

```diff
@@ -51,15 +51,21 @@ class SecurityCubit extends Cubit<SecurityState> with HydratedMixin<SecurityStat
   Future<bool> testPin(String pin) async {
     final String? storedPin = await keyChain.read(pinCodeKey);
     if (storedPin == null) {
-      _setLockState(LockState.locked);
-      throw SecurityStorageException();
+      _logger.warning('PIN not found in storage but state indicates enabled');
```

Comment:
Also here, I don't think we should clear the pin on this function. Do we really need to clear it?